DigitalLearningSolutions.Web/Views/Frameworks/Developer/_SignpostingPaginationNav.cshtml

```diff
@@ -0,0 +1,45 @@
+﻿@using DigitalLearningSolutions.Web.ViewModels.Frameworks
+@model CompetencyResourceSignpostingViewModel
+
+<nav class="nhsuk-pagination" role="navigation" aria-label="Pagination">
+  <form method="post" role="search" asp-controller="Frameworks" asp-action="GoToPage">
+    <div class="nhsuk-pagination__list">
+      <div class="pagination-button-container">
+        <div class="nhsuk-pagination-item--previous" @(Model.Page == 1 ? "hidden" : "")>
+          <button class="nhsuk-pagination__link nhsuk-pagination__link--prev" type="submit"
```

Comment:
This button should be converted to a link to call a Get method and should have search string and page number as route values, e.g.:
```
asp-route-search="@Model.SearchString"
asp-route-page="@(Model.Page-1)"
```